specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations-authoring.json

```diff
@@ -0,0 +1,2703 @@
+      "required": [
+        "result",
+        "jobId",
+        "createdDateTime",
+        "lastUpdatedDateTime",
+        "expirationDateTime",
+        "status",
+        "errors"
+      ],
+      "type": "object",
+      "properties": {
+        "result": {
+          "description": "Represents Training Tasks Detailed Result.",
+          "$ref": "#/definitions/TrainingJobTaskResultObject"
+        },
+        "jobId": {
+          "description": "Gets or sets the jobId.",
+          "type": "string"
+        },
+        "createdDateTime": {
+          "format": "date-time",
+          "description": "Gets or sets the created time.",
+          "type": "string"
+        },
+        "lastUpdatedDateTime": {
+          "format": "date-time",
+          "description": "Gets or sets the last updated time.",
+          "type": "string"
+        },
+        "expirationDateTime": {
+          "format": "date-time",
+          "description": "Gets or sets the expiration time.",
+          "type": "string"
+        },
+        "status": {
+          "description": "Gets or sets the job status.",
+          "enum": [
+            "unknown",
+            "notStarted",
+            "running",
+            "succeeded",
+            "failed",
+            "cancelled",
+            "cancelling",
+            "partiallyCompleted"
+          ],
+          "type": "string"
+        },
+        "errors": {
+          "description": "Gets or sets the errors.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/JobErrorObject"
+          }
+        }
+      }
+    },
+    "TrainingJobTaskResultObject": {
+      "required": [
+        "trainedModelLabel",
+        "trainStatus"
+      ],
+      "type": "object",
+      "properties": {
+        "trainedModelLabel": {
+          "description": "Represents trained model label.",
+          "type": "string"
+        },
+        "trainStatus": {
+          "description": "Represents model train status.",
+          "$ref": "#/definitions/TrainingJobStatusResultStatusObject"
+        },
+        "evaluationStatus": {
+          "description": "Represents model evaluation status.",
+          "$ref": "#/definitions/TrainingJobStatusResultStatusObject"
+        }
+      }
+    },
+    "TrainingJobStatusResultStatusObject": {
+      "required": [
+        "percentComplete",
+        "elapsedTime"
+      ],
+      "type": "object",
+      "properties": {
+        "percentComplete": {
+          "format": "int32",
+          "description": "Represents progress percentage.",
+          "type": "integer"
+        },
+        "elapsedTime": {
+          "description": "Represents the elapsed time.\n            This string is in the format hh:mm:ss",
+          "type": "string"
+        }
+      }
+    },
+    "TrainingJobStatusObjectList": {
+      "required": [
+        "jobs",
+        "nextLink"
+      ],
+      "type": "object",
+      "properties": {
+        "jobs": {
```

Comment:
Mentioned above as well, but applies here and several instances below. For LRO result objects, your array should always be called "value". If you can't change it in the service now, remember you can use `x-ms-client-name` to rename it in generated code.